src/interest.sol

```diff
@@ -28,21 +28,21 @@ contract Interest is Math {
     function compounding(uint chi, uint speed, uint rho, uint pie) public view returns (uint, uint) {
         require(now >= rho, "tinlake-math/invalid-timestamp");
         require(chi != 0);
-        uint updatedChi = rmul(rpow(speed, now - rho, ONE), chi);
+        uint updatedChi = chargeInterest(chi ,speed, rho);
         uint chi_ = safeSub(updatedChi, chi);
         return (updatedChi, rmul(pie, chi_));
     }
 
     // @notice This function updates chi
-    // @param chi Accumulated interest rate over time
+    // @param bearingAmount is the interest bearing amount
```

Comment:
I don't think bearingAmount makes sense. It should be either `amount` or `interestBearingAmount`, just `bearingAmount` is not really used in common english.

```suggestion
    // @param bearingAmount is the interest bearing amount
```